pkgs/applications/misc/sweethome3d/default.nix

```diff
@@ -3,16 +3,16 @@
 , fetchurl
 , makeWrapper
 , makeDesktopItem
-# sweethome3d 6.5.2 does not yet fully build&run with jdk 9 and later?
-, jdk8
-, jre8
+, jdk
+, jre
```

Comment:
Jre and jdk are the same thing:
https://github.com/NixOS/nixpkgs/blob/1a4d722e81aa9156a3028f6b3846d2156b3a4f76/pkgs/top-level/all-packages.nix#L16441-L16448

Reply:
Fixed